src/lib/GHCIDebug.ts

```diff
@@ -102,7 +102,12 @@ export class GHCIDebug {
     if (typeof breakpoint === 'string') {
       this.run(`:break ${breakpoint}`)
     } else {
-      this.run(`:break ${breakpoint.file} ${breakpoint.line}`)
+      this.run(`:show modules`)
+        .then(modules => {
```

Comment:
It boils down to personal preference, but I would prefer `await` instead of `then`.